src/views/Scenario/Scenario.js

```diff
@@ -301,7 +301,7 @@ const Scenario = (props) => {
   };
   return (
     <>
-      <Backdrop open={showBackdrop}>
+      <Backdrop open={showBackdrop} style={{ zIndex: '10000' }}>
```

Comment:
I don't like it. Maybe add a comment saying "Hey, this is ugly, please fix me better."

Reply:
There aren't many ways to fix it, the only improvement we can do right now is extract the style and use a className, would it be ok for you ?
Note: even with mui v5, I can't find any `z-index` defined in the [Backdrop component](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Backdrop/Backdrop.js), so we may have to use the `sx` prop to define a custom style anyway...

Reply:
OK, it seems MUI, for some good reason, doesn't define `z-index` for Backdrop like other ones.
It's hard to tell if we should specify it each time, or globally, in an override file.

For the time being, I accept this fix :)